ogc-example/src/test/scala/geotrellis/server/HillshadeSpec.scala

```diff
@@ -0,0 +1,59 @@
+package geotrellis.server
+
+import com.azavea.maml.ast.{FocalHillshade, RasterLit}
+import com.azavea.maml.error.Interpreted
+import com.azavea.maml.eval.{Interpreter, Result}
+
+import geotrellis.proj4.LatLng
+import geotrellis.raster.{MultibandTile, ProjectedRaster, Raster, RasterExtent, TargetCell}
+import geotrellis.raster.io.geotiff.{AutoHigherResolution, GeoTiff}
+import geotrellis.store._
+import geotrellis.raster.resample._
+import geotrellis.vector.Extent
+import cats.data.Validated.{Invalid, Valid}
+
+import scala.reflect.ClassTag
+
+import org.scalatest.FunSpec
+import org.scalatest._
+
+class HillshadeSpec extends FunSpec with Matchers {
+  implicit class TypeRefinement(self: Interpreted[Result]) {
+    def as[T: ClassTag]: Interpreted[T] = self match {
+      case Valid(r) => r.as[T]
+      case i @ Invalid(_) => i
+    }
+  }
+
+  // https://github.com/geotrellis/geotrellis-server/issues/150
+  describe("HillshadeSpec") {
+    ignore("RasterSource reproject hillshade") {
```

Comment:
Ignore, since it is read from the S3 catalog. Mb we can think of a better test later (?)

Reply:
Ha. I was just about to comment that this test needs credentials. Beat me to it. 

In reality, doesn't this test belong in MAML or GeoTrellis since it doesn't have an GT server components at all? Perhaps one of those repos is better setup for these types of tests.

Reply:
Probably should live eventually in GeoTrellis; wanted to capture it at least somewhere.

Reply:
👍 perhaps make an issue in locationtech/geotrellis? We might never get to it, but then its at least documented somewhere that its something we wanted to do...